src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSTaskTemplate.java

```diff
@@ -576,7 +576,7 @@ public String getDisplayName() {
     /**
      * This merge does not take an into consideration the child intentionally setting empty values for parameters like "entrypoint" - in fact
      * it's not uncommon to override the entrypoint of a container and set it to blank so you can use your own entrypoint as part of the command.
-     * What's really needed is a "MergeStrategy BinaryOperator ECSTaskTemplate that's user selectable.
+     * What's really needed is a "MergeStrategy `BinaryOperator<ECSTaskTemplate>` that's user selectable.
```

Comment:
it was that way before. the javadocs check on release failed due to that
you can check with ` mvn org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar`

Reply:
Ok, strange. This is what I tried to merge in.